dataplane/standalone/saiserver/routing.go

```diff
@@ -175,6 +175,62 @@ func newRoute(mgr *attrmgr.AttrMgr, dataplane routingDataplaneAPI, s *grpc.Serve
 	return r
 }
 
+// CreateRouteEntry creates a new route entry.
+func (r *route) CreateRouteEntry(ctx context.Context, req *saipb.CreateRouteEntryRequest) (*saipb.CreateRouteEntryResponse, error) {
+	rReq := &dpb.AddIPRouteRequest{
+		Route: &dpb.Route{
+			Prefix: &dpb.RoutePrefix{
+				Prefix: &dpb.RoutePrefix_Mask{
+					Mask: &dpb.IpMask{
+						Addr: req.GetEntry().GetDestination().GetAddr(),
+						Mask: req.GetEntry().GetDestination().GetMask(),
+					},
+				},
+				VrfId: req.GetEntry().GetVrId(),
+			},
+		},
+	}
+
+	// TODO(dgrau): Implement CPU actions.
+	switch req.GetPacketAction() {
+	case saipb.PacketAction_PACKET_ACTION_DROP:
+		fallthrough
```

Comment:
I think using fallthrough can be a bit confusing here, why not group the cases?

```
case saipb.PacketAction_PACKET_ACTION_DROP, 
     saipb.PacketAction_PACKET_ACTION_TRAP, 
     saipb.PacketAction_PACKET_ACTION_DENY:
     rReq.Route.Action = dpb.PacketAction_PACKET_ACTION_DROP
```

Reply:
done